packages/experimental/Avatar/src/Avatar.types.ts

```diff
@@ -0,0 +1,101 @@
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { ImageProps, ViewProps, ImageURISource, TextProps, ColorValue } from 'react-native';
+import { IBackgroundColorTokens, IForegroundColorTokens } from '@fluentui-react-native/tokens';
+
+export const avatarName = 'Avatar';
+
+export type AvatarSize = 'size8' | 'size24' | 'size32' | 'size40' | 'size48' | 'size56' | 'size72' | 'size100' | 'size120';
```

Comment:
This is okay for initial check in, but we should align these size props between win32 and AppleOS